### ISSUE TITLE
Add icons (3): mobile preview devices

### ISSUE THREAD

**Issue by folletto** [Automattic/gridicons#61]
I've been sketching an interface exploration with different screen breakpoints, so I tried to use Gridicons as well instead of dashicons and I think we need some extra icons there too.

Context:
- The [gridicons-phone.svg](https://github.com/Automattic/gridicons/blob/master/svg-min/gridicons-phone.svg) is nice, but feels too abstract, "just a rounded square".
- The [gridicons-tablet.svg](https://github.com/Automattic/gridicons/blob/master/svg-min/gridicons-tablet.svg) is abstract too, but also turned in a different direction. This makes its usage more limited.

To be also in sync with the picker in the customizer, we can introduce three icons that are meant to be together. 

Here's an initial idea:

![screen shot 2015-07-20 at 14 56 02](https://cloud.githubusercontent.com/assets/4389/8777423/d5133fb4-2eef-11e5-9c27-aec41e82ec94.png)

Basically:
1. Introduces the screen/computer icon, with a shape that matches the tablet one, just with different bevels.
2. Add a "button" to both tablet and phone so they are more clearly distinctive.
